internal/loadbalancers/infrastructure/manager.go

```diff
@@ -0,0 +1,166 @@
+package infrastructure
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	lbaas "github.com/equinix/metal-cli/internal/loadbalancers/api/v1"
+)
+
+const ProviderID = "loadpvd-gOB_-byp5ebFo7A3LHv2B"
+
+var LBMetros = map[string]string{
+	"da": "lctnloc--uxs0GLeAELHKV8GxO_AI",
+	"ny": "lctnloc-Vy-1Qpw31mPi6RJQwVf9A",
+	"sv": "lctnloc-H5rl2M2VL5dcFmdxhbEKx",
+}
+
+type Pools map[int32][]Target
+
+type Target struct {
+	IP   string
+	Port int32
+}
+
+type Manager struct {
+	client         *lbaas.APIClient
+	metro          string
+	projectID      string
+	tokenExchanger *TokenExchanger
+}
+
+func NewManager(metalAPIKey, projectID, metro string) *Manager {
```

Comment:
In CPEM, the `Manager` type was intended to abstract away the concerns in that particular application, where we need to reconcile a desired load balancer state in k8s with what exists in the world.  In the CLI, there's no reconciliation, so we should use the LB client directly the same way we do for Metal.

Reply:
I copied the directory wholesale. Can remove it from this PR.